### PR TITLE
xds: make log levels consistent

### PR DESCRIPTION
**Pull request by ramaraochavali**
Log levels between cds and lds are inconsistent in the way resource additions are logged. cds logs them at debug and lds logs them at info. This PR makes it consistent. If it is intentionally done like that, I am happy to close this PR

Risk Level: Low
Testing: N/A
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

File: source/common/upstream/cluster_manager_impl.cc [hunk 1]
```diff
@@ -529,7 +529,7 @@ bool ClusterManagerImpl::addOrUpdateCluster(const envoy::api::v2::Cluster& clust
     init_helper_.addCluster(*cluster_entry->cluster_);
   } else {
     auto& cluster_entry = warming_clusters_.at(cluster_name);
-    ENVOY_LOG(info, "add/update cluster {} starting warming", cluster_name);
+    ENVOY_LOG(debug, "add/update cluster {} starting warming", cluster_name);
```
Comment (ramaraochavali) [inline source/common/upstream/cluster_manager_impl.cc]: In the above if block, these are logged at debug level, so making it consistent with that.

File: source/server/lds_api.cc [hunk 2]
```diff
@@ -39,7 +39,7 @@ void LdsApiImpl::onConfigUpdate(
   // with the same address as a listener that is to be removed. Do not change the order.
   for (const auto& removed_listener : removed_resources) {
     if (listener_manager_.removeListener(removed_listener)) {
-      ENVOY_LOG(info, "lds: remove listener '{}'", removed_listener);
+      ENVOY_LOG(debug, "lds: remove listener '{}'", removed_listener);
```
Comment (ramaraochavali) [inline source/server/lds_api.cc]: In cds, these are logged at debug level - so making it consistent with that.

**Comment by lambdai** [comment]
Honestly I didn't pay enough attention to the log prior to this PR.
Personally I think added/removed/warmed are readable for both envoy developers and sysadmin. I prefer leaving these to info. So is the cds.

WDYT?

**Comment by ramaraochavali** [comment]
@lambdai Sure. I have no strong opinion on whether it should be `info` or `debug` - but was looking for consistency. I can change them to info.

**Comment by ramaraochavali** [comment]
Ok. I changed the cds logs to be `info` so that they are consistent with `lds`. I have retained the `cluster_manager_impl` logs at `debug` as cds info level logs would be sufficient for cluster additions/removals.

**Comment by lambdai** [comment]
/lgtm
@ramaraochavali  Thank you so much! I believe we would benefit from the change when I debug in the short future!

**Comment by ramaraochavali** [comment]
@mattklein123 This is stuck with azure pipeline, can you PTAL if this makes sense?

**Pull request merged**